qml/pages/SettingsPage.qml

```diff
@@ -46,15 +45,29 @@ Page {
             TextSwitch {
                 checked: appSettings.sendByEnter
                 text: qsTr("Send message by enter")
-                description: qsTr("Send your message by pressing the enter key")
+                description: qsTr("Send your message by pressing the enter key.")
                 automaticCheck: false
                 onClicked: {
                     appSettings.sendByEnter = !checked
                 }
             }
 
-            VerticalScrollDecorator {}
+            SectionHeader {
+                text: qsTr("Appearance")
+            }
+
+            TextSwitch {
+                checked: appSettings.showStickersAsImages
+                text: qsTr("Show stickers as images")
+                description: qsTr("Normally stickers don't have a background and aligned like text, but you can change that and display them like regilar images.")
```

Comment:
Please don't put valuations ("normally") in descriptive texts and check your spelling ("regilar"). Better: "Show background for stickers and align them centrally like images"

Reply:
Done.